data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/unapproachable_east/una_equip.lst

```diff
@@ -57,30 +57,48 @@ Silver Sign	TYPE:Magic.Wondrous.Sign.Consumable	MODS:REQUIRED		COST:5000	WT:0.01
 # Runescars for Runescarred Berserker
 # 1st level
 Runescar of Cure Moderate Wounds	SPROP:Cure 2d8+ % HP|(min((classlevel("Runescarred Berserker")),30))	TYPE:Magic.Consumable	PRESPELL:1,Cure Moderate Wounds	PRECLASS:1,Runescarred Berserker=1
+#DG todo - cannot add ASPECT to equipment, JIRA'd, may workaround by having equipment add an ability that then adds the aspect...
+#ASPECT:CombatBonus|+% luck bonus to hit/damage for 1 minute (Runescar of Divine Favour)|min(min((classlevel("Runescarred Berserker"))/3,1),3)
```

Comment:
Why are we electing to have ASPECT comments sitting on equipment? Either grant the ability with the ASPECT tag, swap out for a QUALITY tag, or figure out how to implement the INFO tag since that seems to be the eventual direction with text output tags.